handler/src/main/java/io/netty/handler/ssl/JdkSslClientContext.java

```diff
@@ -256,22 +258,22 @@ public JdkSslClientContext(File trustCertChainFile, TrustManagerFactory trustMan
         }
     }
 
-    JdkSslClientContext(X509Certificate[] trustCertChain, TrustManagerFactory trustManagerFactory,
+    JdkSslClientContext(X509Certificate[] trustCertCollection, TrustManagerFactory trustManagerFactory,
                         X509Certificate[] keyCertChain, PrivateKey key, String keyPassword,
                         KeyManagerFactory keyManagerFactory, Iterable<String> ciphers, CipherSuiteFilter cipherFilter,
                         ApplicationProtocolConfig apn, long sessionCacheSize, long sessionTimeout) throws SSLException {
         super(ciphers, cipherFilter, toNegotiator(apn, false), ClientAuth.NONE);
-        ctx = newSSLContext(trustCertChain, trustManagerFactory, keyCertChain, key, keyPassword,
+        ctx = newSSLContext(trustCertCollection, trustManagerFactory, keyCertChain, key, keyPassword,
                             keyManagerFactory, sessionCacheSize, sessionTimeout);
     }
 
-    private static SSLContext newSSLContext(X509Certificate[] trustCertChain, TrustManagerFactory trustManagerFactory,
-                                            X509Certificate[] keyCertChain, PrivateKey key, String keyPassword,
-                                            KeyManagerFactory keyManagerFactory, long sessionCacheSize,
-                                            long sessionTimeout) throws SSLException {
+    private static SSLContext newSSLContext(X509Certificate[] trustCertCollection,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00107)